package.json

```diff
@@ -28,6 +29,7 @@
     "logger-sharelatex": "^2.2.0",
     "mongodb": "^3.6.0",
     "p-map": "^4.0.0",
+    "request": "^2.88.2",
```

Comment:
Should `request` also be removed from `devDependencies`?

Reply:
Thanks, yes. Done via https://github.com/overleaf/docstore/pull/102/commits/a98ab5bbfb805a225bc9f0c9f471407b8ca91b18